### ISSUE TITLE
Re-check with the include and exclude energy use history

### ISSUE THREAD

**Issue by baipai** [codeforboston/home-energy-analysis-tool#126]
Although the design below didn't showcase it, the original design/plans from a while ago included a visual where, upon entering this page, the system would automatically select the 'include' data based on calculations from the uploaded CSV file in the previous step. Users could then review the included data and manually exclude any if they desired, or if the system was unable to identify it.
<br>
![2 10 Step 9 - Ready for dev](https://github.com/codeforboston/home-energy-analysis-tool/assets/11348330/f8e2dd8d-164e-498b-b56b-ec0d5bd2d9c4)

During the discussion on 12/26, @plocket  raised the question of whether it makes sense to indicate whether the data is automatically selected or manually adjusted by users.

<br>
The original flow is designed with the goal of aiding users in reviewing automatically included or excluded data. It does not emphasize marking the differences between automatic and manual data inputs, as the primary focus is on achieving 'the end result,' aligning more with human behavior.
But, if it's essential for users to tell apart auto and manual data for decision-making, we might need a different approach.

</br>
</br>

Here are some areas/questions that we might need to clarify, and there could be additional points to consider. I sense there might be a difference in perspective between the engineering and design sides in this case. The functionality can occur in the background without necessarily being displayed in the design/front.

- How do automatic or manual data inputs affect users' decision-making? 
- Would marking them differently produce different results?
- What's the goal for the users here?

**Comment by plocket** [comment]
I think that's a good point. Is it actually useful to the user to be aware of which settings are theirs vs which changes were automatic. After all, if the app needs to reset to automatic settings, it can just re-run the calculation. Maybe I was off-base on this.

**Comment by baipai** [comment]
On 1/2, here were several data conversations with the entire team (design, JavaScript, and rules engine), leading to the need to display distinct heating usage and non-heating usage (other usage) in the visual with override sections.

During the design meetings on 1/2 and 1/9, various options were explored, including color coding, icons, dropdown menus, and implementing a default button.


Aspects to consider include:

- What are the users' goals with this setting?
- What objectives are we helping users achieve through this feature?
- How will the default setting appear in the absence of any adjustments?

**Comment by baipai** [comment]
> I think that's a good point. Is it actually useful to the user to be aware of which settings are theirs vs which changes were automatic. After all, if the app needs to reset to automatic settings, it can just re-run the calculation. Maybe I was off-base on this.

No, you are bringing out the right point. It's good to see the conversation happening among the entire team rather than on an individual level.

**Comment by ethanstrominger** [comment]
Implemented
